indico/modules/events/notes/controllers.py

```diff
@@ -94,17 +96,18 @@ def _process_POST(self, render_mode, source):
             logger.info('Note %s modified by %s', note, session.user)
             self.event.log(EventLogRealm.participants, LogKind.change, 'Minutes', 'Updated minutes',
                            session.user, data=note.link_event_log_data)
-        return '', 204
+        return EventNoteSchema().dump(note.current_revision), 200
```

Comment:
```suggestion
        return EventNoteSchema().jsonify(note.current_revision)
```

that's already the default. also let's use the proper json content-type